### ISSUE TITLE
esm-tools 6: long waiting time for job submission with esm_runscripts

### ISSUE THREAD

**Issue by fernandadialzira** [esm-tools/esm_tools#560]
With esm tools 6 I am experiencing a long waiting time to submit an experiment either in check mode [esm_runscripts -c], or for a real submission. The waiting time ranges from 15 min up to 2 hours, mostly during prepcompute phase.

Machine: ollie
Esm-tools branch: wiso-pico
Model: awiesm-2.1-gamma

The path to the experiment is /work/ollie/fmatos/mis3/run_configs_testing/glac1d21_classic.yaml

**Comment by mandresm** [comment]
An important note here, is that this only happens when you use release 6 from `direnv` with the specifical `.envrc` you are using. I could not reproduce this error on a normal situation where I use `ESM-Tools` from the local environments. So my suspicion is that the problem is in the `.envrc`.

**Comment by pgierz** [comment]
....that sounds like it is linked to my "construction site". The HPC friends have not yet said that there are any strange disk issues. A few questions:

1. Is it intermittent?
1. Is is only if you have `direnv` on?
1. Can we get a copy of the actual copy commands being done, to try them by hand?

**Comment by pgierz** [comment]
For question 1., Fernanda has often told me that she has slow start times, so I do not believe it is intermittent....

**Comment by fernandadialzira** [comment]
In terms of the `.envrc` file, at least now esm tools is installed in the environment itself not in the `~/.local/`. Now regarding the questions:

1. The lower waiting time I had was 15 min, so it is definitely not an intermittent issue...
2. I am not sure if I understood the question correctly, but to my knowledge direnv is always on. I only turn `use_venv` to `False` in the yaml file
3. Perhaps would it help if I try another experiment just to have screenshots of the job submission process?

**Comment by pgierz** [comment]
For number 2, can you check please what happens if you rename the `.envrc` file to something else? In that case, the shell will not start the local environment. I am not sure if that will have any other side-effects (e.g. your tools might be on a different version suddenly). But, then we have one less thing to have as a problem candidate....

```
$ mv ${PROJECT_BASE}/.envrc ${PROJECT_BASE}/.envrc_OFF
... log off and on again ...
... start your run and see if takes long 
```

**Comment by fernandadialzira** [comment]
I changed the name of `.envrc` and adapted the run script to the path without setting the environment. The submission job took 15 minutes which for me is good given the number of files to be read. So perhaps we continue without the `.envrc`? No error messages regarding the submission process were there. I also checked the version and path for esm_toos/runscripts/master and they are all pointing to the esm_tools folder under mis3 with version 6.
